### ISSUE TITLE
abnormal NADPH oxidase activity

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#2292]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia):  abnormal NADPH oxidase activity

>2. Synonyms (e.g., absent spleen):  abnormal NAD(P)H:oxygen oxidoreductase activity

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

"an abnormal level of the enzyme that catalyzes the reaction: NAD(P)H + H+ + O2 = NAD(P)+ + hydrogen peroxide."  PMID:19406829

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.  PMID:19406829

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal enzyme/coenzyme level (MP:0005319).  

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name):  RGD:sjwang

**Comment by sbello** [comment]
Added term 'Abnormal NAD(P)H oxidase activity' MP:0020445 under parent abnormal enzyme/coenzyme activity. I used the activity definition and placement rather than level  as this seems to fit what the paper was talking about. I used NAD(P)H in the primary term name as this matches what GO has.

**Comment by slaulederkind** [comment]
I forgot to add that the "increased..." and "decreased..." children terms of the requested term are also needed.  Thanks.

**Comment by sbello** [comment]
MP:0020446 - increased
MP:0020447 - decreased
